### PR TITLE
Add azure datalake gen 2 docs

### PR DESCRIPTION
**Pull request by bakobako**

### CODE REVIEW

**Comment by webrouse** [comment]
ZD ticket 18743

**Comment by webrouse** [comment]
Hi @bakobako 

Screenshots must capture the full page (excluding modals) so that the user can see where he is.

See:
https://keboola.atlassian.net/wiki/spaces/KB/pages/82935879/Public+documentation#Component-Docs
image

Example:
http://help.keboola.com/components/extractors/storage/aws-s3/

File: components/extractors/storage/index.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ Extractors import data from external sources and integrate it to the Keboola Con
 The following extractors allow access to data from generic storage services:
 
 - [AWS S3](/components/extractors/storage/aws-s3) --- imports CSV files from multiple AWS S3 buckets into multiple tables with additional postprocessing.
+- [Azure Datalake Gen2](/components/extractors/storage/azure-datalake-gen2) --- imports CSV files from Azure Datalake Gen2 into multiple tables with additional postprocessing.
```
Comment (webrouse) [inline components/extractors/storage/index.md:12]: I suggest removing ` with additional postprocessing.` , it's unnecessarily long.

**Comment by bakobako** [comment]
Made the changes

**Comment by webrouse** [comment]
@bakobako mozes to prosim rebasnut (` git rebase -i origin/master`) a vyhodit commit `removed criteo branch from this one`, ktory generuje konflikt?

Spravil by som to ja, ale nemozem pushnut do tvojho repa.

**Pull request merged**